docs/05-terraform.md

```diff
@@ -37,13 +37,17 @@ This Terraform setup is designed to deploy and manage a Managed kdb Insights env
 New user please continue and follow this section - Users with existing infrastructure, please skip to our [existing infrastructure section](#deploying-with-terraform-for-users-with-existing-infrastructure).
 
 1. (Optional) If you have an HDB you want to migrate to FinSpace, replace the dummy HDB in `TorQ-Amazon-FinSpace-Starter-Pack/hdb`.
-2. Move into the `TorQ-Amazon-FinSpace-Starter-Pack/terraform-deployment/deployments` directory; this will be the Terraform working directory from which you should run all `terraform` commands.
-3. Modify variables inside the `terraform.tfvars` file, such as region name, environment name, database name. You can modify it by replacing the variable name inside of `"Name"`. For example, For the variable on `role-name`, you can change the variable name by replacing `"finspace-role"`.
-4. (Optional) If you have changed the database name from the default `finspace-database` to any other names, please also edit the `env.q` file, changing the database name to the new variable that you have set in line 19.
-5. Run `aws configure` in the terminal to set up your access key and secret key from your AWS account. This is needed to connect to your account and use the Terraform deployment. Check our resource link for more instructions on how to find your access key and secret key.
-6. From your Terraform working directory which is `TorQ-Amazon-FinSpace-Starter-Pack/terraform-deployment/deployments`, run `terraform init`.
-7. If initialized without error, run `terraform plan`. This will show all resources set to be created or destroyed by Terraform.
-8. Run `terraform apply` to execute this plan. The initial deployment can take approximately 45 minutes, and connection losses can cause errors with deployment, so it's a good idea to run this in `nohup`. (Using `nohup` might lead to a higher cost of operating the codes if you are using Terraform from a cloud environment.) Example nohup run: `nohup terraform apply -auto-approve > terraform_apply.log 2>&1 &`. 
+2. Move into the `TorQ-Amazon-FinSpace-Starter-Pack/terraform-deployment/deployments` directory; within there should be two sub-directories : `date-par` and `int-par`. 
+  If you want your finspace with Managed Kdb Insights system to use date partitions use `date-par`
+  If you want your system to perform intra-day writedowns with int partitions use `int-par` 
+  Choose one to be the the Terraform working directory from which you should run all `terraform` commands.
+3. (Optional) symlink your hdb and your zipped source code to your working directory
```

Comment:
Why? What is this step for?

Reply:
I just added this because the zip_file_path and hdb-path configured in terraform.tfvars would otherwise be something like "../../../../code.zip" and "../../../inthdb|datehdb/hdb" respectfully. Thus, it's an option for cleanliness in the configurations, but definitely not a mandatory step